.github/ISSUE_TEMPLATE/bug_report_agent.md

```diff
@@ -2,14 +2,11 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: bug
+labels: bug, agent
```

Comment:
This expands the issue templates so that there are separate templates for

* Bug -> agent
* Bug -> server
* Enhancement -> agent
* Enhancement -> Server

I wish we could be a little smarter about it so that we could repurpose, but I think it's a side-effect of our repository structure more than it's a problem with the issue template capabilities.